domains.js

```diff
@@ -7,4 +7,32 @@ const domainsList = [
     site: "https://howmanyfavoursdoesmattowecrystal.com/",
```

Comment:
Can you check the first sites as well to see if their costs are accurate? We might've just pulled them out of thin air 😳